package ch10_ex02;

public class Throw01 {

	public static void main(String[] args) {
		System.out.println("안녕");
		Exception e1 = new Exception("내가 Error 만들었다");
		
		System.out.println("e1 1");
		throw e1;
		System.out.println("e1 2");
	}

}
